utils/stdx/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "stdx"
+version = "0.0.0"
+authors.workspace = true
+edition.workspace = true
+publish = false
```

Comment:
this'll break because vm-logic is public=true. I think this means that this crate wants to be called `near-stdx` and be `publish=true`